data/coalition/coalition missions.txt

```diff
@@ -1551,49 +1553,62 @@ mission "Saryd University Lecture"
+				`	"Humans have official contact with the Quarg, but humans and Quarg don't really intermix."`
+				`	"Humans know about the Quarg, but most people know little about them."`
+					goto quarg
 			label hai
-			`	"Aliens besides the Quarg, are humans in contact with?"`
+			`	"What is the extent of contact that humans have with other aliens?"`
 			choice
-				`	"Yes, humans are in contact with the Hai. They live to the north of human space, and we can visit them through a wormhole."`
+				`	"Humans are in contact with the Hai. Some of them even live in Hai space. They live to the north of human space, and we can visit them through a wormhole."`
 					goto "know"
 				`	"A few humans know about the Hai, but there's no official acknowledgment of their existence. They live to the north of human space, and we can visit them through a wormhole."`
 					goto "know"
-				`	"No, the only aliens humans know about are the Quarg."`
-					goto "sad"
-			label sad
-			`	At this several in the audience sound disappointed.`
+				`	"Humans have official contact with the Quarg, but the Quarg don't trade technology with humanity."`
+				`	"Humans have official contact with the Quarg, but humans and Quarg don't really intermix."`
+				`	"Humans know about the Quarg, but most people know little about them."`
+					goto "quarg"
+			label quarg
+			`	The audience response is muted. You get the impression that you haven't told them anything they don't already know.`
 				goto nextquestion
 			label know
-			`	Several members of the audience are delighted. "Cooperation between different species, it is important to foster," says the Saryd who asked the question. An Arach asks you to point out where the Hai are on a galactic map, and you do so.`
+			`	Several members of the audience are delighted. "Ah, the Hai, we have heard of," says the Heliarch. "Tell us more of them, could you?"`
+			`	As you open your mouth to speak, the Heliarch cuts you off. "Enough questions on the Hai, we have." The crowd looks disappointed at this development.`
 			label nextquestion
 			branch war
 				has "FWC End: completed"
 			`	The Arach selects another, this time a Kimek. "All of human space, united politically is it?"`
 			choice
 				`	"No, human space is split between the Republic in the north and the Free Worlds in the south."`
 				`	"No, human space is split between the Republic in the west, the Free Worlds in the south, and the Syndicate in the east. The Syndicate is technically part of the Republic, but they have a lot of autonomy."`
-			`	There are murmurs in the crowd, and several begin to write down notes on various devices."`
+			`	As you answer the question, your translator begins to cut out. Repeating your answer does no good, as it contiunes to fizzle out as you talk. You look over to the Heliarch, who eyes the curious Kimek with suspicion. "Apologize for the malfunction, I must," says the Heliarch. "The next question, answer it please."`
```

Comment:
Why did they cut it out here? Just want to know

Reply:
My reasoning was that the Heliarchs may not want civilians to know that humans can function fine without a united government. I don't know if that's a misinterpretation of what Heliarchs would be concerned about, though.

Reply:
It's fine, I just wasn't sure on the reason.